default/AI/FreeOrionAI.py

```diff
@@ -209,7 +209,9 @@ def generateOrders():  # pylint: disable=invalid-name
     if turn == 1:
         declare_war_on_all()
         human_player = fo.empirePlayerID(1)
-        fo.sendChatMessage(human_player,  '%s Empire (%s):\n"Ave, Human, morituri te salutant!"' % (empire.name, aggression_name))
+        greets = UserStringList("AI_FIRST_TURN_GREETING")
```

Comment:
I suggest to add a `random.shuffle(greets)` after that, so the order gets randomized (don't want to get the same greetings on every map :wink:).